stdlib/universe/flux_test_gen.go

```diff
@@ -152928,2715 +152928,6 @@ var FluxTestPackages = []*ast.Package{&ast.Package{
-											Source: "fn",
-											Start: ast.Position{
-												Column: 101,
-												Line:   55,
-											},
-										},
-									},
-									Name: "fn",
-								},
-								Separator: nil,
-								Value: &ast.Identifier{
-									BaseNode: ast.BaseNode{
-										Comments: nil,
-										Errors:   nil,
-										Loc: &ast.SourceLocation{
-											End: ast.Position{
-												Column: 112,
-												Line:   55,
-											},
-											File:   "limit_test.flux",
-											Source: "t_limit",
-											Start: ast.Position{
-												Column: 105,
-												Line:   55,
-											},
-										},
-									},
-									Name: "t_limit",
-								},
-							}},
-							Rbrace: nil,
-							With:   nil,
-						},
-						Lparen: nil,
-						Rparen: nil,
-					},
-					Lparen: nil,
-					Params: []*ast.Property{},
-					Rparan: nil,
-				},
-			},
-			BaseNode: ast.BaseNode{
-				Comments: nil,
-				Errors:   nil,
-				Loc: &ast.SourceLocation{
-					End: ast.Position{
-						Column: 114,
-						Line:   55,
-					},
-					File:   "limit_test.flux",
-					Source: "test _limit = () => ({input: testing.loadStorage(csv: inData), want: testing.loadMem(csv: outData), fn: t_limit})",
-					Start: ast.Position{
-						Column: 1,
-						Line:   55,
-					},
-				},
-			},
-		}},
-		Eof: nil,
-		Imports: []*ast.ImportDeclaration{&ast.ImportDeclaration{
-			As: nil,
-			BaseNode: ast.BaseNode{
-				Comments: nil,
-				Errors:   nil,
-				Loc: &ast.SourceLocation{
-					End: ast.Position{
-						Column: 17,
-						Line:   4,
-					},
-					File:   "limit_test.flux",
-					Source: "import \"testing\"",
-					Start: ast.Position{
-						Column: 1,
-						Line:   4,
-					},
-				},
-			},
-			Path: &ast.StringLiteral{
-				BaseNode: ast.BaseNode{
-					Comments: nil,
-					Errors:   nil,
-					Loc: &ast.SourceLocation{
-						End: ast.Position{
-							Column: 17,
-							Line:   4,
-						},
-						File:   "limit_test.flux",
-						Source: "\"testing\"",
-						Start: ast.Position{
-							Column: 8,
-							Line:   4,
-						},
-					},
-				},
-				Value: "testing",
-			},
-		}},
-		Metadata: "parser-type=rust",
-		Name:     "limit_test.flux",
-		Package: &ast.PackageClause{
-			BaseNode: ast.BaseNode{
-				Comments: nil,
-				Errors:   nil,
-				Loc: &ast.SourceLocation{
-					End: ast.Position{
-						Column: 22,
-						Line:   1,
-					},
-					File:   "limit_test.flux",
-					Source: "package universe_test",
-					Start: ast.Position{
-						Column: 1,
-						Line:   1,
-					},
-				},
-			},
-			Name: &ast.Identifier{
-				BaseNode: ast.BaseNode{
-					Comments: nil,
-					Errors:   nil,
-					Loc: &ast.SourceLocation{
-						End: ast.Position{
-							Column: 22,
-							Line:   1,
-						},
-						File:   "limit_test.flux",
-						Source: "universe_test",
-						Start: ast.Position{
-							Column: 9,
-							Line:   1,
-						},
-					},
-				},
-				Name: "universe_test",
-			},
-		},
 	}, &ast.File{
```

Comment:
I am seeing some failures in the test when ran locally with `narrowTransformationLimit: true`

```
--- FAIL: TestFluxEndToEnd (12.61s)
    --- FAIL: TestFluxEndToEnd/contrib/bonitoo-io/tickscript (0.22s)
        --- FAIL: TestFluxEndToEnd/contrib/bonitoo-io/tickscript/deadman_empty (0.08s)
            flux_test.go:146: found 1 tables that were not empty
```

Reply:
I added a new test `n=0` to `limit_test.go` since this tickscript test was the only place in our tests where we hit this specific case.

The test was fixed (again) with offset/state manipulation in `.processChunk`.